lib/Controller/DocumentController.php

```diff
@@ -1242,7 +1242,7 @@ private function putRelative($fileId, $owner, $editor, $suggested) {
 	 * @return null|\OCP\Files\File
 	 */
 	private function getFileHandle($fileId, $owner, $editor) {
-		if ($editor && $editor != '') {
+		if ($editor) {
```

Comment:
If `$editor` is the empty string, then `$editor` will evaluate to `false` in the first condition, so the 2nd condition adds no value.